geomstats/_backend/tensorflow/__init__.py

```diff
@@ -763,10 +763,26 @@ def cumprod(a, axis=None):
     return tf.math.cumprod(a, axis=axis)
 
 
-def tril(m, k=0):
-    if k != 0:
-        raise NotImplementedError("Only k=0 supported so far")
-    return tf.linalg.band_part(m, -1, 0)
+# (sait) there is tf.experimental.tril (we can use it once it moves to stable)
```

Comment:
Use the TODO(sait) syntax: it will automatically create an issue on GitHub, which will be a good reminder for us to look at it.

Reply:
@SaitejaUtpala